src/Compilers/Core/Portable/SourceFileResolver.cs

```diff
@@ -62,9 +62,9 @@ public SourceFileResolver(ImmutableArray<string> searchPaths, string baseDirecto
                     }
 
                     var value = kv.Value;
-                    if (value == null || value.Length == 0)
+                    if (value == null)
                     {
-                        throw new ArgumentException(CodeAnalysisResources.EmptyValueInPathMap, nameof(pathMap));
```

Comment:
I think I did.  It was the change in src/Compilers/Core/Portable/CodeAnalysisResources.Designer.cs